aderyn_core/src/framework/foundry.rs

```diff
@@ -209,29 +210,3 @@ fn get_matching_output_files(
         .cloned()
         .collect()
 }
-
```

Comment:
Reason for removing this one?

Reply:
I think this is fine, it seems to test foundry more than aderyn's interpretation of foundry output files